docs/D-Modularity/pointers.md

```diff
@@ -185,83 +189,91 @@ a is 5
+    <td align="center"><code>0x0012FF7C</code></td>
+    <td align="center"><code>0x0012FF6C</code></td>
+    <td align="center"><code>0x0012FF88</code></td>
+    <td align="center"><code>0x0012FF84</code></td> </tr>
 
 <tr><td>&nbsp;</td>
     <td>&nbsp;</td>
     <td>&nbsp;</td>
-    <td align="center" bgcolor="#FFFF99">5</td>
-    <td align="center" bgcolor="#FFFF99">6</td> </tr>
-
-<tr><td align="center" bgcolor="#FFFF99">5</td>
-    <td align="center" bgcolor="#FFFF99">6</td>
-    <td align="center" bgcolor="#FFFF99">?</td>
-    <td align="center">5</td>
-    <td align="center">6</td> </tr>
-
-<tr><td align="center" bgcolor="#FFFF99">5</td>
-    <td align="center" bgcolor="#FFFF99">6</td>
-    <td align="center" bgcolor="#FFFF99">5</td>
-    <td align="center">5</td>
-    <td align="center">6</td> </tr>
-
-<tr><td align="center" bgcolor="#FFFF99">6</td>
-    <td align="center" bgcolor="#FFFF99">6</td>
-    <td align="center" bgcolor="#FFFF99">5</td>
-    <td align="center">5</td>
-    <td align="center">6</td> </tr>
-
-<tr><td align="center" bgcolor="#FFFF99">6</td>
-    <td align="center" bgcolor="#FFFF99">5</td>
-    <td align="center" bgcolor="#FFFF99">5</td>
-    <td align="center">5</td>
-    <td align="center">6</td> </tr>
-
-<tr><td align="center">6</td>
-    <td align="center">5</td>
-    <td align="center">5</td>
-    <td align="center" bgcolor="#FFFF99">5</td>
-    <td align="center" bgcolor="#FFFF99">6</td> </tr>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>6</code></td> </tr>
+
+<tr><td align="center"><code>5</code></td>
+    <td align="center"><code>6</code></td>
+    <td align="center">?</td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>6</code></td> </tr>
+
+<tr><td align="center"><code>5</code></td>
+    <td align="center"><code>6</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>6</code></td> </tr>
+
+<tr><td align="center"><code>6</code></td>
+    <td align="center"><code>6</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>6</code></td> </tr>
+
+<tr><td align="center"><code>6</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>6</code></td> </tr>
+
+<tr><td align="center"><code>6</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>5</code></td>
+    <td align="center"><code>6</code></td> </tr>
 </table>
 
 The hexadecimal values below the variable identifiers are their addresses in memory.
 
-> Note that the addresses of `a` and `b` in `internal_swap()` are different from those in `main()`. 
+:::note
+
+Note that the addresses of `a` and `b` in `internal_swap()` are different from those in `main()`. 
+
+:::
```

Comment:
Don't repeat the "Note" for admonitions... in this case start with "The addresses..."

Reply:
Fixed!